src/main/java/com/marklogic/batch/rdbms/ApplicationProperties.java

```diff
@@ -0,0 +1,72 @@
+package com.marklogic.batch.rdbms;
+
+import com.marklogic.client.DatabaseClient;
+import com.marklogic.client.DatabaseClientFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.jdbc.datasource.DriverManagerDataSource;
+import org.springframework.stereotype.Component;
+
+import javax.sql.DataSource;
+import java.util.List;
+
+@Component
+public class ApplicationProperties {
```

Comment:
This should have class comments to explain what the scope of "application properties" is, and why command line arguments aren't considered part of it. 

I think a common question will be - why are some things specified in application.properties, but other things are specified on the command line? This class should answer that.